com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -89,6 +89,7 @@ public static bool IsInitialized
         /// <summary>
         /// Collection of float properties (indexed by a string).
         /// </summary>
+        [Obsolete("Academy.FloatProperties has been deprecated, use Academy.GetSideChannel<>() instead.")]
```

Comment:
Should we just remove it? Otherwise we'll be stuck with it for all of 1.x

Reply:
I am okay with removing it. But we are not 100% sure next release is going to be 1.x are we?